README.md

```diff
@@ -128,8 +128,20 @@ the `request` and `response` value will be `null`.
 The specification is validated daily by the [client-flight-recorder](https://github.com/elastic/clients-flight-recorder) project.
 The validation result can be found [here](https://github.com/elastic/clients-flight-recorder/blob/dev/recordings/types-validation/types-validation.md).
 
-If you need to fix the specification and then validate the result again, you need to
-perform the following commands:
+### Validate the specification in your machine
+
```

Comment:
Please add the following preamble:
_The following steps only apply if you don't have `~/.elastic/github.token` in place_